docs/releases/v13/version-v13.0.0.md

```diff
@@ -92,7 +92,27 @@ When a record is shared, its inclusion requests will be also accessible. There i
 
 To be completed
 
-### Collections
+### Communities
+
+#### Themed communities
+
+Communities can now have their own theming with a custom font and colors, which apply to all community pages including records and requests. Below is an example of two themed communities on Zenodo.
+
+![A default community and two themed communities on Zenodo](imgs/themed-communities.png)
+
+Themed communities benefit from a custom homepage, defined via HTML template in `<instance>/templates/themes/<theme>/invenio_communities/details/home/index.html`.
+
+#### Subcommunities
+
+It is now possible to create heirarchical relationships between communities, allowing for departments, subject areas and other structures to be represented via related communities. Records from the "child" community are automatically indexed in the "parent" community, allowing all the records of the children to be browsed in the parents. The communities are also bidirectionally linked so that it is easy to navigate between both.
+
+Having subcommunities also enables the **Browse** page, which lists all the subcommunities and collections (see below) of that community.
+
+!!! note
+
+    Currently communities can only have one level of heirarchy (i.e., no grand-child communities) and communities can only have one parent community. The former restriction would be more easily overridden in your instance if required.
```

Comment:
```suggestion
    Currently communities can only have one level of hierarchy (i.e., no grand-child communities) and communities can only have one parent community. The former restriction would be more easily overridden in your instance if required.
```

Also "would be more easily overridden" as in "it can easily be overridden in your instance"? Or, in the future we will make it overridable on a per instance basis? If former, probably best to word it as such.

Reply:
Via the programmatic API, one can infinitely nest communities; however, it has been decided that we don't want to officially support this, as it greatly complicates permissions (for both developers and end-users to understand).

Right now, we have the following building blocks for nesting content:

- Communities
  - Sub-communities
    - Collections Trees
      - Collections (at this point infinitely nestable actually, but just for organizing content browsing/discovery, and not access/permissions)